src/Eviden.VirtualGrocer/Server/Eviden.VirtualGrocer.Web.Server.csproj

```diff
@@ -38,7 +38,10 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Extensions.AspNetCore.Configuration.Secrets" Version="1.2.2" />
+    <PackageReference Include="Azure.Identity" Version="1.10.0" />
     <PackageReference Include="Azure.Search.Documents" Version="11.4.0" />
+    <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.5.0" />
```

Comment:
We may not need this library now that we're not using the secrets client?

Reply:
Will remove 👍🏼